bin/inx-chronicle/src/main.rs

```diff
@@ -153,6 +153,19 @@ impl HandleEvent<Report<InxListener>> for Launcher {
             Err(e) => match &e.error {
                 ActorError::Result(e) => match e.deref() {
                     InxListenerError::Inx(e) => match e {
+                        InxError::ConnectionError(_) => {
+                            let wait_interval = self.inx_connection_retry_interval;
+                            log::info!("Retrying INX connection in {} seconds.", wait_interval.as_secs_f32());
+                            tokio::time::sleep(wait_interval).await;
```

Comment:
Are you sure about this? This will block the event handler for `wait_interval` seconds. It's probably better to spawn a task here so waiting for the time to pass doesn't block other events.

Reply:
@Alexandcoats is going to do that in a separate PR. Thanks!